Kwf/Update/20150612ComponentCacheTag.sql

```diff
@@ -0,0 +1,2 @@
+#tags: kwc
+ALTER TABLE  `cache_component` ADD  `tag` VARCHAR( 255 ) CHARACTER SET ASCII COLLATE ascii_bin NOT NULL AFTER  `value`;
```

Comment:
column should be indexed
could take very long when table is large, maybe should be truncated before